`make generate-tiles-pg` without editing `.env`

Currently, `make generate-tiles-pg` always produces an error

```
[Error: SQLITE_CANTOPEN: unable to open database file] {
  errno: 14,
  code: 'SQLITE_CANTOPEN'
}
make: *** [generate-tiles-pg] Error 1
```

In [this Slack thread](https://osmus.slack.com/archives/CH6GFAFRC/p1609608482047400), @TomPohys wrote:
> Hi, did you uncomment line 55 in `.env` ? I got this error when I forgot to uncomment `EXPORT_DIR`.

In addition to not being obvious, such editing of the `.env` file breaks the ability to run `quicktime.sh`.

--- a/.env
+++ b/.env
@@ -51,7 +51,5 @@ COPY_CONCURRENCY=10
 
 # Variables for generate tiles using PGquery
 PGHOSTS_LIST=
-# Uncomment when used PGquery
-#EXPORT_DIR=/import
 NO_GZIP=1
 USE_KEY_COLUMN=1

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -53,6 +53,7 @@ services:
     volumes:
       - .:/tileset
       - ./data:/import
+      - ./data:/export
       - ./build/sql:/sql
       - ./build:/mapping
       - ./cache:/cache

